### PR TITLE
Web development - Courses & Coding playgrounds

### PR DESCRIPTION
**Pull request by Virtual4087**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

### Description: 

Added few resources for full stack development and also added a online coding playground.

### Resource URL: 

https://www.theodinproject.com
https://www.youtube.com/watch?v=nu_pCVPKzTk
www.programiz.com

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: Web Development/README.md [hunk 1]
```diff
@@ -172,6 +172,16 @@ this course has two sections:
     <td><a href="https://communitytaught.org/class/all">100devs</a></td>
     <td>100Devs is a diverse and inclusive community of 1000s of participants in a training program run by Leon Noel. It is 100% free and requires absolutely no previous technical experience. This is not a coding bootcamp, this is a how-to-get Full Stack Web Dev job bootcamp</td>
   </tr>
+
+  <tr>
+    <td><a href="https://part-time.learnhowtoprogram.com/tracks">The Odin Project</td>
```
Comment (jfmartinz) [inline Web Development/README.md:177]: The url you provided seems not align with The Odin Project. Double check it and change it to something like this

Comment (jfmartinz) [reply Web Development/README.md:177]: I will merged once you completed this issue. Thanks :)

**Review by Virtual4087 — COMMENTED** [review]
I'm sorry, I'll change it right away

**Review by jfmartinz — APPROVED** [review]
Thank you @Virtual4087! ❤️

**Pull request merged**